assert/assertions.go

```diff
@@ -426,6 +426,37 @@ func BeLessOrEqualTo[T Ordered](t testing.TB, actual T, expected T, opts ...Opti
 	}
 }
 
+// BeInRange reports a test failure if the value is not within the specified range (inclusive).
```

Comment:
```suggestion
// BeBetween reports a test failure if the value is not within the specified range (inclusive).
```

What about this? Just a suggestion, I'm unsure what would be the clearest